dot/state/block.go

```diff
@@ -183,6 +192,28 @@ func (bs *BlockState) GenesisHash() common.Hash {
 	return bs.genesisHash
 }
 
+// ValidateTransaction validates transaction
+func (bs *BlockState) ValidateTransaction(e types.Extrinsic) (*transaction.Validity, error) {
```

Comment:
where is this used? doesn't look like `core` uses it, seems weird for it to be in the block state anyways

Reply:
Removed. I added for author module RPC. But it seems the author module doesn't use it.